### ISSUE TITLE
import googleapis as npm module fails (1.3-beta.11)

### ISSUE THREAD

**Issue by jschlieber** [meteor/meteor#6438]
I tried to import the [googleapis](https://github.com/google/google-api-nodejs-client) npm module with the new meteor 1.3 es6 import feature, but it fails:

```
Error: Cannot find module './aes'
     at require (packages/modules-runtime/.npm/package/node_modules/install/install.js:77:1)
     at node_modules/node-forge/js/forge.js:31:1
     at Array.map (packages/es5-shim/.npm/package/node_modules/es5-shim/es5-shim.js:450:1)
     at Module.module.exports (node_modules/node-forge/js/forge.js:30:1)
     at defineFunc (node_modules/node-forge/js/forge.js:47:1)
     at node_modules/node-forge/js/forge.js:90:1
     at define (node_modules/node-forge/js/forge.js:15:1)
     at define (node_modules/node-forge/js/forge.js:54:1)
     at node_modules/node-forge/js/forge.js:59:1
     at meteorInstall.node_modules.node-forge.js.forge.js (node_modules/node-forge/js/forge.js:92:1)
```

To reproduce: 

1. `meteor create --release 1.3-beta.11 reproduce-this-issue`
2. `cd reproduce-this-issue`
3. `npm init`
4. `npm install --save googleapis`
5. Create file `server/issue.js`
6. First line in `server/issue.js`:

```javascript
import googleapis from 'googleapis';
```

Then run `meteor`.

**Comment by zol** [comment]
Please try this again after the next beta.

**Comment by servermeta** [comment]
I'm having a similar issues. Packages I tried:

1) google-spreadsheet : works with Npm.require, not import or require.

https://forums.meteor.com/t/require-vs-import-in-meteor-1-3/19204

2) sweetalert : works with import, but doesn't load css, so I have to manually put it in client folder

Beta 15, 14, 13 doesn't start for me. Beta 12 has testing bug, so it always show testing interface. All betas show this probelm.

**Comment by servermeta** [comment]
seems linked to issue #6408  #6376 and others

Many packages from npm are not working, or working funny, I could list a ton. 

IS this going to be fixed after 1.3? it means that 1.3 will not have true NPM support.

**Comment by benjamn** [comment]
This should be fixed in `beta.16`, thanks to #6398.
